shared/store/saga-monitor.js

```diff
@@ -0,0 +1,517 @@
+// Based off of. Keybase stuff marked with KB START/END
```

Comment:
This is a copy of the file, but added transforms and whatnot so we don't output most info (like arguments to things). We can expand whats shown later